judgels-frontends/raphael/src/routes/jophiel/profiles/single/submissionHistory/SubmissionHistoryPage/SubmissionHistoryPage.tsx

```diff
@@ -59,6 +61,7 @@ class SubmissionHistoryPage extends React.Component<SubmissionHistoryPageProps,
       <SubmissionsTable
         submissions={submissions.page}
         userJid={this.props.userJid}
+        sessionUserJid={this.props.sessionUserJid}
```

Comment:
Can we just pass `userJid={this.props.sessionUserJid}`

Reply:
Yes, turns out I can do this. Done.